decidim-core/app/controllers/concerns/decidim/amendments_controller.rb

```diff
@@ -71,7 +98,13 @@ def validate(form)
         on(:invalid) do
           flash[:alert] = t("created.error", scope: "decidim.amendments")
           params[:amend][:emendation_fields] = params[:amend][:emendation_fields]
-          redirect_to new_amend_path(amendable_gid: @form.amendable_gid)
+          case params[:action]
```

Comment:
I don't understand this, why do we need `Decidim::Amendable::Validate`?

Reply:
For nothing, now that the code have been refactored thanks to your suggestions. Let me know if it can be improved:
https://github.com/decidim/decidim/pull/3985/commits/6389395706b6157f07e8d7adb0e4d5bccea63321